2021-22/2021-12-02_50p.py

```diff
@@ -29,13 +29,19 @@ def goto(target_x, target_y, reverse):
     delta_y = y2 - y1
     distance = math.sqrt(delta_x**2 + delta_y**2)     # pythagorean theorem
     if ( delta_x ):
-        directio = math.atan(delta_y / delta_x) * 180 / math.pi
+        directio = - math.atan(delta_y / delta_x) * 180 / math.pi
     else:
         directio = 90
 
-    #directio = math.atan(delta_y / delta_x) * 180 / math.pi  
+    if ( delta_x < 0 ):
+        directio = directio + 180
+
```

Comment:
atan() returns only +90 to -90 degrees, but the robot operations in 360 degrees range. Negative delta_x implies it's turning left!